src/gurt/telemetry.c

```diff
@@ -1824,6 +1853,269 @@ d_tm_add_metric(struct d_tm_node_t **node, char *metric, int metric_type,
 	return rc;
 }
 
+/**
+ * Creates histogram counters for the given node.  It calculates the
+ * extents of each bucket and creates counters at the path specified that
+ * correspond to each bucket required.  The name of each counter created is
+ * given by the bucket number.  The bucket number and range of each bucket
+ * is stored as metadata for each counter.
+ *
+ * \param[in]	node		Pointer to a node with a metric of type duration
+ *				or gauge.
+ * \param[in]	path		Path name of the metric specified.
+ *				by \a node.  Can be an arbitrary location.
+ *				However, specifying the full path allows this
+ *				function to create counters underneath the given
+ *				node.
+ * \param[in]	num_buckets	Specifies the number of buckets the histogram
+ *				should have.
+ * \param[in]	initial_width	The number of elements in the first bucket
+ * \param[in]	multiplier	Applied to the second .. nth bucket to
```

Comment:
I'm of the opinion that if we apply a multiplier then the second bucket should be the width of the first bucket times the multiplier, this isn't the case in the commit message example and this comment alludes to the fact that the first two buckets will be of the same width and the doubling will only start from the third bucket which seems inconsistent.

Reply:
Done.